### ISSUE TITLE
Connecting a chain to the software

### ISSUE THREAD

**Issue by onurt91** [viveris/jtag-boundary-scanner#14]
Hello, I am having problems connecting 2 boards on one chain and trying to control it with the software.

The chain looks like this:
PC (JLINK) -> D1: XMC4700(QFP) -> D2: XMC4800(BGA).   (each chip consists of 2 devices)

I can only control device D1 in EXTEST mode, otherwise, nothing else is working. 
When I select D2 and compare the TDI and TDO lines of D1, they are different, which is why D2 is not getting the correct information.

When using a single device ( PC (JLINK) -> D1: XMC4700(QFP) ), SAMPLE and EXTEST mode are working correctly.

Does the software support chaining of multiple devices or is the problem more likely on my side? 
(I am not an expert in Boundary Scan, but looks like the D1 is not put into bypass mode? )

Thank you for your help

**Comment by jfdelnero** [comment]
Yes it should support devices chaining. if possible please retry with the latest version.
